osquery/filesystem/windows/fileops.cpp

```diff
@@ -375,18 +375,16 @@ static Status hasAccess(HANDLE handle, mode_t mode) {
                                        GROUP_SECURITY_INFORMATION |
                                        DACL_SECURITY_INFORMATION;
 
-  status =
-      ::GetUserObjectSecurity(handle, &security_info, nullptr, 0, &sd_size);
-  if (status || (!status && ::GetLastError() != ERROR_INSUFFICIENT_BUFFER)) {
+  status = GetUserObjectSecurity(handle, &security_info, nullptr, 0, &sd_size);
+  if (status || (!status && GetLastError() != ERROR_INSUFFICIENT_BUFFER)) {
     return Status(-1, "GetUserObjectSecurity get SD size error");
   }
 
   std::vector<char> sd_buffer;
```

Comment:
Consolidate this info `std::vector<char> sd_buffer(sd_size, '\0');` similar to the others